### DISCUSSION TITLE
Yarn has occupied my macOS home dir… how to change?

### DISCUSSION THREAD

**Discussion by justindonofrio-ow** [Homebrew/discussions#5212] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: bb753a6c48575ae018d7ad925d7762d59647fff7
Last commit: 3 days ago
Core tap JSON: 18 Mar 15:48 UTC
Core cask tap JSON: 18 Mar 15:48 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.3.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.3
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hi everyone,

I installed yarn via Homebrew and it's working fine. 

However, my home directory is now a mess, with yarn slapping a whole bunch of stuff in there:

~/node_modules
~/node_modules/yarn
~/package-lock.json
~/package.json
~/yarn.lock

I'd really like to change this up, and have yarn stick all this stuff literally anywhere else. 

Since I installed Yarn via Homebrew, I thought I'd make this my first stop in looking for a how-to, as I'm coming up a bit short trying to find a clear one.

Thanks so much!

**Comment by SMillerDev** [comment]
Did you run something like `yarn init` in your home directory?

**Reply by justindonofrio-ow** [reply]
Hi! I did not run init specifically, though now that I think of it, I may have ran a post-homebrew command or two whilst in my home directory. -___-
